pkg/cli/completion/completion.go

```diff
@@ -28,7 +28,7 @@ func Run(ctx *cli.Context) error {
 func genCompletionScript(shell string) (string, error) {
 	var completionScript string
 	if shell == "bash" {
-		completionScript = fmt.Sprintf(`#! /bin/bash
```

Comment:
Unlike the linked post about python, bash installs are not nearly as volatile in their install location. /bin/bash is the de facto location for the bash shell. Does this fix a specific issue you have on a specific OS?

Reply:
First of all, this shebang is corrupt. And `/bin/bash` is missing in some OS like FreeBSD.

Reply:
... we don't support FreeBSD so I'm not sure why it matters. 

Also note that the current value while technically wrong in practice works fine. Since the #! line is invalid, the current interpreter is used... which presumably is bash since you are asking for bash completions.

```console
brandond@dev01:~$ . <(echo -e "#! /bad/invalid\n echo foo")
foo
```

Reply:
I'm happy to fix the corrupted line.